platform/view/services/comm/host/rest/websocket/multiplexed_provider.go

```diff
@@ -186,15 +186,15 @@ func (c *multiplexedClientConn) readIncoming() {
 			sc.reads <- streamEOF
 		}
 		err := c.Conn.Close()
-		logger.Debugf("client connection closed: %v", err)
+		logger.Infof("Client connection closed: %v", err)
 	}()
 	var mm MultiplexedMessage
 	for {
 		//c.writeMu.Lock()
```

Comment:
can these comments be removed? Or accompanied by an explanation why they're commented out?